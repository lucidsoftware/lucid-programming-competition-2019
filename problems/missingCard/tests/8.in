19 3
1 13 9 11 6 15 14 5 3 4 7 12 16 8 18 17 19 2
11 4 12 15 10 14 7 2 8 5 17 3 1 19 6 9 13 16
5 18 10 11 7 19 4 15 6 8 17 16 2 3 12 1 13 9