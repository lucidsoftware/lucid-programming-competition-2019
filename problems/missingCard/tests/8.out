10
18
14